package.nls.json

```diff
@@ -107,7 +107,8 @@
     "jupyter.command.jupyter.collapseallcells.shorttitle": "Collapse",
     "jupyter.command.jupyter.addcellbelow.title": "Add Empty Cell to File",
     "jupyter.command.jupyter.scrolltocell.title": "Scroll Cell Into View",
-    "jupyter.command.jupyter.createnewnotebook.title": "Create New Blank Notebook",
+    "jupyter.command.jupyter.createnewnotebook.title": "New Jupyter Notebook",
```

Comment:
I could revert this back to "Create New Blank Notebook" if folks thing that works better. But basing it off of the same pattern as here: https://github.com/microsoft/vscode-github-issue-notebooks/blob/5baec6d14c60d47cbf779bf1c04f14a51b0555c4/package.json#L74 made more sense to me since there are multiple types of notebooks and the "Blank" didn't seemed needed to say when creating new. 

The long title shows up in the command palette, the short title shows up in the File New... menu.

Reply:
@greazer @claudiaregio tagging for notice as you might have an opinion on this rename.

Reply:
@IanMatthewHuff I agree with you. Following the GitHub Issues example below and adding 'Jupyter Notebook' as the entry here makes a lot of sense. 
![image](https://user-images.githubusercontent.com/19276747/134113657-75c123f3-1019-4d69-9c32-3d27d205eb4b.png)

Am I understanding correctly that here the command palette option would be renamed to 'New Jupyter Notebook'?
My only concern is for users who go autopilot in command palette and type "Create" would no longer find that and may get confused. Would you be opposed to the option being "Create New Jupyter Notebook"?